### PR TITLE
Run formatter on Mix.Tasks.New

### PR DESCRIPTION
**Pull request by mhinz**

### CODE REVIEW

File: lib/mix/lib/mix/tasks/new.ex [hunk 1]
```diff
@@ -147,65 +153,74 @@ defmodule Mix.Tasks.New do
...
+      Mix.raise("Module name #{inspect(name)} is already taken, please choose another name")
     end
   end
 
   defp check_directory_existence!(path) do
-    if File.dir?(path) and not Mix.shell.yes?("The directory #{inspect(path)} already exists. Are you sure you want to continue?") do
-      Mix.raise "Please select another directory for installation"
+    if File.dir?(path) and
+         not Mix.shell().yes?(
+           "The directory #{inspect(path)} already exists. Are you sure you want to continue?"
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/new.ex]: Can you please move the message to a variable that is defined before the `if`? Thanks.

**Comment by josevalim** [comment]
@mhinz I have added one comment and we are good to go! :heart:

File: lib/mix/lib/mix/tasks/new.ex (hunk 1 above)
Comment (mhinz) [reply lib/mix/lib/mix/tasks/new.ex]: Done. I hope `msg` is a valid variable name here, since it's a rather short function.

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**